dashboard/templates/base.html

```diff
@@ -1,18 +1,13 @@
 {% load static %}
-{% load macros %}
 {% load tz %}
 
 <!DOCTYPE html>
-<!-- Use this as a "macro", no default title set -->
-{% macro title %}
-    {% block title %}{% endblock %}
-{% endmacro %}
 <html lang="en" xmlns="http://www.w3.org/1999/html">
 <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>{% use_macro title %}</title>
+    <title>{% block title %}{% endblock %}</title>
```

Comment:
It looks like this was what we originally had before adding macros in 2018.
https://github.com/tl-its-umich-edu/my-learning-analytics/pull/174/files

I don't remember now why macros was used. Is it that this just isn't needed at all with React controlling the title now?

Reply:
Yeah, looks like we just wanted to use the title multiple times before (repeat a block), but we aren't doing that any more with React handling the frontend.